src/devcenter/azext_devcenter/vendored_sdks/devcenter_dataplane/aio/operations/_catalog_items_operations.py

```diff
@@ -65,25 +137,21 @@ async def get(
             401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
         }
         error_map.update(kwargs.pop('error_map', {}))
-        api_version = "2022-03-01-preview"
+        api_version = "2022-11-11-preview"
```

Comment:
Can we put this in a constants file ?

Reply:
These files under the vendored_sdks folder will be auto-generated and written over, so leaning towards not having a constants file for these